src/cookies.zig

```diff
@@ -63,6 +101,23 @@ pub const Cookie = struct {
         };
     }
 
+    /// vec must be large enough for the largest cookie (4 + attributes)
+    pub fn writeVec(c: Cookie, vec: []std.posix.iovec_const) !usize {
```

Comment:
Same here with changing it to return  `[]iovec_const` if you agree with that change for Cookie.